### PR TITLE
Newton raphson fix error

### PR DESCRIPTION
**Pull request by AmilaMadushan**
### Describe your change:

* [ ] Add an algorithm?
* [X] Fix a bug or typo in an existing algorithm?
* [ ] Documentation change?

### Checklist:
* [X] I have read [CONTRIBUTING.md](https://github.com/TheAlgorithms/Python/blob/master/CONTRIBUTING.md).
* [X] This pull request is all my own work -- I have not plagiarized.
* [X] I know that pull requests will not be merged if they fail the automated tests.
* [X] This PR only changes one algorithm file.  To ease review, please open separate PRs for separate algorithms.
* [X] All new Python files are placed inside an existing directory.
* [X] All filenames are in all lowercase characters with no spaces or dashes.
* [X] All functions and variable names follow Python naming conventions.
* [X] All function parameters and return values are annotated with Python [type hints](https://docs.python.org/3/library/typing.html).
* [X] All functions have [doctests](https://docs.python.org/3/library/doctest.html) that pass the automated testing.
* [X] All new algorithms have a URL in its comments that points to Wikipedia or other similar explanation.
* [X] If this pull request resolves one or more open issues then the commit message contains `Fixes: #{$ISSUE_NO}`.

### CODE REVIEW

**Comment by AmilaMadushan** [comment]
fixed the error in https://github.com/TheAlgorithms/Python/pull/6332

**Comment by Mariya346** [comment]
Added

File: maths/newton_raphson.py [hunk 1]
```diff
@@ -23,11 +23,17 @@ def newton_raphson(f, x0=0, maxiter=100, step=0.0001, maxerror=1e-6, logsteps=Fa
...
+    privestimate = a
+    nextestimate = 0
+    
     f1 = lambda x: calc_derivative(f, x, h=step)  # noqa: E731  Derivative of f(x)
     for _ in range(maxiter):
         if f1(a) == 0:
             raise ValueError("No converging solution found")
         a = a - f(a) / f1(a)  # Calculate the next estimate
+        nextestimate = a
+        error = abs(privestimate-nextestimate)
```
Comment (poyea) [inline maths/newton_raphson.py:35]:
```suggestion
        error = abs(privestimate - nextestimate)
```

File: maths/newton_raphson.py [hunk 2]
```diff
@@ -23,11 +23,17 @@ def newton_raphson(f, x0=0, maxiter=100, step=0.0001, maxerror=1e-6, logsteps=Fa
     a = x0  # set the initial guess
     steps = [a]
     error = abs(f(a))
+    privestimate = a
+    nextestimate = 0
+
```
Comment (poyea) [inline maths/newton_raphson.py:28]:
```suggestion

```

**Comment by cclauss** [comment]
In preparation for that denial-of-service attack that is [Hacktoberfest](https://hacktoberfest.com), I am closing all pull requests that have failing tests.

**Pull request closed without merging**